src/autoyast-rnc/rules.rnc

```diff
@@ -74,6 +76,8 @@ hostaddress    = element hostaddress     { MAP, (match_text & match_type?) }
 hostid         = element hostid          { MAP, (match_text & match_type?) }
 karch          = element karch           { MAP, (match_text & match_type?) }
 linux          = element linux           { MAP, (match_text & match_type?) }
+installed_product         = element installed_product         { MAP, (match_text & match_type?) }
+installed_product_version = element installed_product_version { MAP, (match_text & match_type?) }
```

Comment:
it looks like it break rest of indentation

Reply:
Not really, I simply decided to indent these lines in the same way, as the element names are longer than the rest.